### PR TITLE
fix(ui): Update fixed colors (#2945)

### PR DESCRIPTION
**Pull request by hervyt**
# Context
> Update all fixed colors to fit with dark/light theme 

## How to test
- Evrything shoud be ok

## Related issues

- Related to #2945 

# Checklist

## Quality

- [X] I consider this work complete and ready for review
- [X] I tested all features and edge cases
- [X] I refactored code where necessary to improve quality
- [X] I made sure my code meets development guidelines
- [X] I performed a self-review of my code
- [ ] I added the `skip-feature-env` label if a feature environment is not needed for this PR

## Testing

- [ ] I wrote test cases for the relevant use cases (unit, integration or e2e tests)
- [ ] (Bug fixes only) I added tests that reproduce and verify the fix

### What tests have been made

- [ ] Unit tests
- [ ] Integration tests
- [ ] E2E tests
- [X] Local manual tests

### CODE REVIEW

File: apps/frontend/eslint.config.mjs [hunk 1]
```diff
@@ -178,6 +178,46 @@ const localI18nRulesPlugin = {
   },
 };
 
+const fixedTailwindColorClassPattern =
```
Comment (carinelebas) [inline apps/frontend/eslint.config.mjs:181]: ❤️

**Review by carinelebas — APPROVED** [review]
